account_analytic_profit_loss_analysis/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Analytic Account Profit and Loss Analysis",
+    "summary": "",
+    "description": """""",
+    "version": "10.0.1.0.0",
+    "category": "Accounting",
+    "website": "https://www.odoo-asia.com/",
+    "author": "Quartile Limited",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "account",
+        "project_parent",
```

Comment:
Add `sale_timesheet`.  `project_project_id` field of sale.order comes from this module.

Reply:
"sale_timesheet" is added to the depends list